From df76d3c7b72e1126af7a8f80b4048d199a88321d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 2 May 2021 07:47:05 +0000
Subject: [PATCH] Add missing Gtk version declaration

Otherwise it fails to run if gtk4 is installed on the system.
---
 griffith | 1 +
 1 file changed, 1 insertion(+)

diff --git a/griffith b/griffith
index f62e719..0b2287b 100755
--- a/griffith
+++ b/griffith
@@ -86,6 +86,7 @@ if len(missing) > 0:
 del missing
 
 # other imports
+gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
 import add
 import config
-- 
2.31.1

